### PR TITLE
[android][devmenu] Fix dev menu not opening on Android

### LINKED ISSUE #7713: Dev menu doesn't show up any more when you shake the device on Android
### Environment

```

  Expo CLI 3.17.18 environment info:
    System:
      OS: macOS 10.15.3
      Shell: 5.7.1 - /bin/zsh
    Binaries:
      Node: 13.8.0 - /usr/local/bin/node
      Yarn: 1.22.0 - /usr/local/bin/yarn
      npm: 6.14.4 - /usr/local/bin/npm
      Watchman: 4.9.0 - /usr/local/bin/watchman
    IDEs:
      Android Studio: 3.6 AI-192.7142.36.36.6200805
      Xcode: 11.4/11E146 - /usr/bin/xcodebuild
    npmPackages:
      expo: ^37.0.6 => 37.0.6 
      react: 16.9.0 => 16.9.0 
      react-native: https://github.com/expo/react-native/archive/sdk-37.0.0.tar.gz => 0.61.4 
      react-navigation: ^4.3.7 => 4.3.7 
    npmGlobalPackages:
      expo-cli: 3.17.18
```

Target: Android

### Steps to Reproduce

Shake device

### Expected Behavior

Should show the new dev menu

### Actual Behavior

Doesn't show anything

### LINKED ISSUE #7775: [Android] Dev menu not opening when app launched directly
- If you launch the app directly via `adb shell am start -a android.intent.action.VIEW -d "exp://127.0.0.1:19000"` the dev menu will not respond to `adb shell input keyevent 82` or `cmd+m`.
- Probably unrelated (because similar things happened on SDK 36 client) but odd - launching Expo client via `adb shell am start -a android.intent.action.VIEW -d "exp://"` hangs indefinitely.
- This is the root cause for https://github.com/expo/expo/issues/7772 which has been temporarily worked around by https://github.com/expo/expo-cli/commit/8839d5d1b1b12aeaea54e897a1b14ffa322daaa9

### PR DESCRIPTION
**Pull request by tsapeta**
# Why

Fixes #7713 
Fixes #7775 
Related #7772 

# How

Shame to admit, but for some unknown reason this change didn't land in #6829 even though I was sure it did 😅 

# Test Plan

Tested by opening an experience by pressing `a` in `expo start` process and then pressing Cmd+M in the emulator.

### CODE REVIEW

**Review by brentvatne — APPROVED** [review]

**Comment by brentvatne** [comment]
let's wait for @esamelson to review before merging but lgtm

**Review by esamelson — APPROVED** [review]

**Pull request merged**
